tests/python_tests/test_generate_api.py

```diff
@@ -133,13 +133,13 @@ def stop_criteria_map():
 
 
 test_cases = [
-    (dict(max_new_tokens=20), 'table is made of'),
-    (dict(max_new_tokens=20), '你好！ 你好嗎？'),
-    (dict(num_beam_groups=3, num_beams=15, num_return_sequences=15, max_new_tokens=20, diversity_penalty=1.0), 'Alan Turing was a'),
-    (dict(num_beam_groups=3, num_beams=15, num_return_sequences=15, max_new_tokens=30, diversity_penalty=1.0), 'Alan Turing was a'),
-    (dict(num_beam_groups=2, num_beams=8, num_return_sequences=8, max_new_tokens=20, diversity_penalty=1.0), 'table is made of'),
-    (dict(num_beam_groups=2, num_beams=8, num_return_sequences=8, max_new_tokens=20, diversity_penalty=1.0), 'The Sun is yellow because'),
-    (dict(num_beam_groups=2, num_beams=8, num_return_sequences=8, max_new_tokens=20, diversity_penalty=1.5), 'The Sun is yellow because'),
+    (dict(do_sample=False, max_new_tokens=20), 'table is made of'),
+    (dict(do_sample=False, max_new_tokens=20), '你好！ 你好嗎？'),
+    (dict(do_sample=False, num_beam_groups=3, num_beams=15, num_return_sequences=15, max_new_tokens=20, diversity_penalty=1.0), 'Alan Turing was a'),
```

Comment:
why do we need it? I assumes that current dict is used to create new GenerationConfig instead of updating existing one

Reply:
It was reverted